README.md

```diff
@@ -139,6 +139,6 @@ I would like to thank Microverse for including this in their curriculum.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT) licensed.
 
```

Comment:
- [ ] I got 404 error while trying to access the MIT license link. I observed that the MIT file extension ( `.md` which typically stands for Markdown) is missing from the href. ℹ️  Markdown is a lightweight markup language that is often used for formatting text, especially in documents and web content.

```suggestion
This project is [MIT](./MIT.md) licensed.

```